src/components/Block/ResultsError.vue

```diff
@@ -0,0 +1,15 @@
+<script setup>
+import { useI18n } from 'vue-i18n';
+const { t } = useI18n();
+</script>
+
+<template>
+  <Block :title="$t('results')" icon="warning" iconClass="text-red">
+    <div>{{ t('resultsError') }}</div>
+    <UiButton class="w-full mt-3">
```

Comment:
Clicking in these area doesn't work 
<img width="352" alt="image" src="https://user-images.githubusercontent.com/51686767/153363807-63ff048b-23ee-46d1-a6e2-e87ca0c3cf1f.png">

Reply:
Oh yea, needs to be nested the other way around of course. Thanks.